docs/pages/docs/rtl-support.mdx

```diff
@@ -0,0 +1,118 @@
+import { Tabs, Tab } from 'nextra/components'
+
+# RTL supoprt
+
+In case you want the `dir` attribute of your HTML node to adjust based on the current locale, you'll need to add some logic.
+
+## From the root layout
+
+```tsx
+// layout.tsx (root layout)
+export default function Layout({ children, params: { locale } }: { children: ReactElement, params: { locale: string } }) => {
+  const dir = new Intl.Locale(locale).getTextInfo().direction
+
+  return (
+    <html lang={locale} dir={dir}>
+      <body>
+        {children}
+      </body>
+    </html>
+  );
+};
```

Comment:
We usually try to remove any excessive markup to make it easier to read (so remove any `;`):
```suggestion
// app/[locale]/layout.tsx
export default function Layout({ children, params: { locale } }: { children: ReactElement, params: { locale: string } }) => {
  const dir = new Intl.Locale(locale).getTextInfo().direction

  return (
    <html lang={locale} dir={dir}>
      <body>
        {children}
      </body>
    </html>
  )
}
```